SpatialGDK/Source/Public/Interop/SpatialTypebindingManager.h

```diff
@@ -9,36 +9,47 @@
 
 #include "SpatialTypebindingManager.generated.h"
 
-enum ERPCType
+enum EComponentType : int32
 {
-	RPC_Client = 0,
-	RPC_Server,
-	RPC_CrossServer,
-	RPC_NetMulticast,
-	RPC_Count
+	TYPE_Invalid = -1,
+
+	// Properties
+	TYPE_Data,
+	TYPE_OwnerOnly,
+	TYPE_Handover,
+
+	// RPCs
+	TYPE_ClientRPC,
+	TYPE_ServerRPC,
+	TYPE_NetMulticastRPC,
+	TYPE_CrossServerRPC,
+
+	TYPE_Count
```

Comment:
When using TYPE_Data as the start of an iteration loop, how about adding a TYPE_Start/TYPE_Begin that points to it?